compiler/test/dotty/tools/dotc/printing/PrintingTest.scala

```diff
@@ -21,18 +21,16 @@ class PrintingTest {
   val testsDir = "tests/printing"
   val options = List("-Xprint:typer", "-color:never", "-classpath", TestConfiguration.basicClasspath)
 
-  private def fileContent(filePath: String): List[String] =
-    if (new File(filePath).exists)
-      Source.fromFile(filePath, "UTF-8").getLines().toList
-    else Nil
-
-
   private def compileFile(path: JPath): Boolean = {
     val baseFilePath  = path.toString.stripSuffix(".scala")
     val checkFilePath = baseFilePath + ".check"
     val byteStream    = new ByteArrayOutputStream()
     val reporter = TestReporter.reporter(new PrintStream(byteStream), INFO)
 
+    val jOutFilePath = Paths.get(baseFilePath + ".check.out")
+    if (Files.exists(jOutFilePath))
+      try { Files.delete(jOutFilePath) } catch { case _: Exception => () }
+
```

Comment:
Will it be better to remove the file in `FileDiff.checkAndDump`?

Reply:
You're right. I can do it in [FileDiff.checkAndDump](https://github.com/lampepfl/dotty/blob/5374d91fc103191455360cdd6a4c69a0c7c50189/compiler/test/dotty/tools/vulpix/FileDiff.scala#L46).

I'll move the code.